Sources/Sentry/SentryUIViewControllerSwizzling.m

```diff
@@ -69,6 +71,15 @@ - (instancetype)initWithOptions:(SentryOptions *)options
 
 - (void)start
 {
+    SentryBinaryImageCache *imageCache = SentryDependencyContainer.sharedInstance.binaryImageCache;
+
+    for (NSString *string in self.inAppLogic.inAppIncludes) {
+        NSString *pathToImage = [imageCache pathForImage:string];
+        if (pathToImage != nil) {
+            [self swizzleUIViewControllersOfImage:pathToImage];
+        }
```

Comment:
`l`: I think maybe logging a warning that we tried to swizzle UIViewControllers for the inAppInclude, but we couldn't find the image path for it, would be excellent.